Packs/GitHub/ReleaseNotes/2_0_29.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Github Event Collector
+
+- Updated the Docker image to: *demisto/fastapi:1.0.0.87576*.
+- Fixed a bug that caused duplicates in the **fetch-github-events**.
```

Comment:
What is **fetch-github-events**?  Is it a command?  a parameter?  an argument?  I did not see it in the yml file.

Reply:
It's a build-in command so it's not in the YML file